python_ta/cfg/__main__.py

```diff
@@ -0,0 +1,55 @@
+import click
+
+from .cfg_generator import generate_cfg
+
+
+def parse_visitor_options(options_str):
+    """Parse comma-separated key=value pairs."""
+    if not options_str:
+        return {}
+
+    options = {}
+
+    if "separate-condition-blocks=" in options_str:
```

Comment:
Overall it isn't very generalizable to search for specific hard-coded option names (this is another form of coupling between the CLI file and the `CFGVisitor` class).

I'm guessing you didn't want to just split on `','` because of the possible commas inside the `functions` value. But, since we're assuming that any commas in a value must be surrounded by quotes, you can implement a custom splitting function of the form "split `options_str` by commas, as long as those commas aren't between two matching quotes".